spec/intl.html

```diff
@@ -25,26 +25,25 @@ <h1><a href="https://tc39.es/ecma402/#sec-time-zone-names">Time Zone Names</a></
     </ins>
 
     <p>
-      The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database.
+      The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database <ins>except as specifically overridden by CanonicalizeTimeZoneName</ins>.
     </p>
 
     <p>
-      All registered Zone and Link names are allowed. Implementations must recognize all such names, and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations. However, the set of combinations of time zone name and language tag for which localized time zone names are available is implementation dependent.
+      All registered Zone and Link names are allowed. Implementations must recognize all such names <ins>(and <strong>only</strong> such names) as built-in</ins>, and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations. However, the set of combinations of time zone name and language tag for which localized time zone names are available is implementation dependent.
     </p>
 
+    <del class="block">
```

Comment:
Looking up built-in time zones in ECMA-262 IsValidTimeZoneName removes the need to define that algorithm in ECMA-402.